database/src/main/resources/db/migration/V22__behandling_saksbehandler.sql

```diff
@@ -0,0 +1,3 @@
+alter table behandling
+    add column if not exists
+        saksbehandletAv text;
```

Comment:
saksbehandler?

Reply:
Som Jacob sier, har vi kalt dette feltet saksbehandler de andre plassene, men jeg kan være enig i at saksbehandletAv leser enda bedre.

Dette kunne vel i teorien vært not null, siden det er en saksbehandler som setter i gang selve saksbehandlingen (og opprettelsen av den). Har vi tenkt noe på hva som skjer dersom fler enn 1 person er inne å saksbehandler samme (søknads)behandling?

Reply:
Jeg *tipper* at det siste ikke vil skje i praksis, siden saksbehandlingsoppgavene blir tildelt til saksbehandlere 1-til-1. Men det betyr jo også at vi evt. burde forhindre det i vår løsning.
Og så vil jeg jo tippe det skjer at en saksbehandler blir syk og ikke kan fortsette, og at oppgaven da flyttes til en annen, men da vil jeg også tippe at den nye saksbehandleren må se på hele saken på nytt.

Men tenker @aryani1 kan følge opp spørsmålet sitt til Agnethe, og bare finne ut hva hvordan dette vil håndteres i praksis.